lib/spack/spack/test/spack_yaml.py

```diff
@@ -125,6 +126,7 @@ def test_yaml_aliases():
         ),
     ],
 )
+@pytest.mark.skipif(sys.platform == "win32", reason="fails on Windows")
```

Comment:
What does the failure on windows look like, is that something we can reasonably address?

Reply:
The unit tests adds newlines on Windows, while is supposed not. My guess is that it may have to do with `\n\r` vs. `\n` but I don't have a Windows machine to try it on.